python/cudf/cudf/core/column/column.py

```diff
@@ -1145,6 +1138,16 @@ def build_column(
             children=children,
             null_count=null_count,
         )
+    elif dtype is cudf.core.dtypes.ListDtype:
```

Comment:
Wouldn't we want to check that `dtype` is an instance of `ListDtype` as opposed to being the class itself?

Reply:
Right you are -- updated to use `is_list_dtype`